src/index.ts

```diff
@@ -845,19 +844,19 @@ async function main() {
     .addCommand(
       new Command("transfer")
         .description("Send tokens from the ICRC wallet to another account.")
-        .option(
+        .requiredOption(
           "--canister-id <canister-id>",
-          "Canister ID (defaults to ICP Ledger)",
+          "ICRC ledger Canister ID",
           tryParsePrincipal
         )
         .requiredOption(
-          "--to <account-identifier>",
+          "--to <account>",
           "ICRC Account",
           tryParseIcrcAccount
         )
         .requiredOption(
           "--amount <amount>",
-          "Amount to transfer in e8s",
+          "Amount to transfer in ledger's base unit",
```

Comment:
Not necessarily for this PR, but would it make sense if this is specified like "1.23 TICRC1" as opposed "123000000 TICRC1" in the future?